.gitlab-ci.yml

```diff
@@ -32,13 +32,13 @@ workflow:
     REDISCLI_AUTH:
       vault:                       cicd/gitlab/parity/REDIS_AUTH@kv
       file:                        false
-    GITHUB_TOKEN:
-      vault:                       cicd/gitlab/parity/ink/GITHUB_TOKEN@kv
+    GITHUB_PR_TOKEN:
```

Comment:
The variable name also appears further down in the file, can you rename it there as well?

Reply:
Ah my bad, mixed it up.